.github/CODEOWNERS

```diff
@@ -14,11 +14,25 @@
 # Root files
 /conftest.py  @thejcannon
 
-# src/python
-/src/python/pants/backend/python/  @thejcannon
-/src/python/pants/backend/experimental/python  @thejcannon
+# Docs
+/docs/markdown/Docker/ @kaos
+/docs/markdown/Using\ Pants/ @kaos
+/docs/markdown/Writing\ Plugins/ @kaos
+
+# Backends
 /src/python/pants/backend/build_files  @thejcannon
```

Comment:
Mind adding trailing slashes to mine?

Also we should find out the nuance of the difference. I assume there is none when the target is a directory. But the trailing slash implies to the reader it is a directory.